sphinx/source/docs/user_guide/tools.rst

```diff
@@ -800,37 +787,36 @@ PointDrawTool
 * name: ``'point_draw'``
 * menu icon: |point_draw_icon|
 
-The ``PointDrawTool`` allows adding, dragging, and deleting point-like
+The ``PointDrawTool()`` allows you to add, drag, and delete point-like
 glyphs (of ``XYGlyph`` type) on one or more renderers by editing the
-underlying ``ColumnDataSource`` data. Like other drawing tools, the
-renderers that are to be edited must be supplied explicitly as a
-list:
+underlying ``ColumnDataSource`` data. Like other drawing tools,
+you must pass the renderers to be edited as a list:
 
 .. code-block:: python
 
     c1 = p.circle('x', 'y', 'width', 'height', source=source)
     r1 = p.rect('x', 'y', 0.1, 0.1, source=source2)
     tool = PointDrawTool(renderers=[c1, r1])
 
-The tool will automatically modify the columns on the data source
+The tool automatically modifies the columns on the data source
 corresponding to the ``x`` and ``y`` values of the glyph. Any
 additional columns in the data source will be padded with the declared
-``empty_value``, when adding a new point. Any newly added points will
-be inserted on the ``ColumnDataSource`` of the first supplied
+``empty_value`` when adding a new point. Any newly added points will
+be inserted in the ``ColumnDataSource`` of the first supplied
 renderer.
```

Comment:
Let me know what you guys think of the latest edits... I struggled on the sections like this one, so if you have any suggestions I'd love to hear them!